pkg/customers/approval.go

```diff
@@ -21,49 +17,16 @@ import (
 	"github.com/go-kit/kit/log"
 )
 
-var (
-	ofacMatchThreshold float32 = func() float32 {
-		if v := os.Getenv("OFAC_MATCH_THRESHOLD"); v != "" {
-			f, err := strconv.ParseFloat(v, 32)
-			if err == nil && f > 0.00 {
-				return float32(f)
-			}
-		}
-		return 0.99 // default, 99%
-	}()
-)
-
-// AddApprovalRoutes contains "back office" admin endpoints used to validate (or reject) a Customer
-// TODO(adam): We need to hide these behind an admin level auth, but we'll write them for now.
-// What about a header like x-admin-id ??
-func AddApprovalRoutes(logger log.Logger, svc *admin.Server, repo CustomerRepository, customerSSNRepo CustomerSSNRepository, ofac *OFACSearcher) {
-	svc.AddHandler("/customers/{customerID}/status", updateCustomerStatus(logger, repo, customerSSNRepo, ofac))
-}
-
 type updateCustomerStatusRequest struct {
```

Comment:
Is there a `client.UpdateCustomerStatus` generated model we can use?

Reply:
updated!